inc/auth.class.php

```diff
@@ -1155,12 +1135,10 @@ static function showSynchronizationForm(User $user) {
             case self::LDAP :
                //Look it the auth server still exists !
                // <- Bad idea : id not exists unable to change anything
-               $sql = "SELECT `name`
-                       FROM `glpi_authldaps`
-                       WHERE `id` = '" . $user->getField('auths_id') . "'
-                             AND `is_active` = 1";
-               $result = $DB->query($sql);
-               if ($DB->numrows($result) > 0) {
+               // SQL query
+               $result = $DB->request('glpi_authldaps', ['FIELDS' => 'name', ['id' => $user->getField('auths_id'), 'is_active' => 1]]);
```

Comment:
As for me, the current doc is against the master, not the 9.1. I had in mind to branch documentation when 9.2 will be released (since the doc is very recent).

It would be OK to consider the doc is for the 9.1; but in that case we'll have to get a development version quite soon; the stable version would miss many parts.

For the dbiterator backport; I guess this is OK, since this is very recent, this is not widely used yet :)